client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getPaperSizes } from '../../lib/pdf-label-utils';
+import Button from 'components/button';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import PaymentMethod, { getPaymentMethodTitle } from './label-payment-method';
+
+class ShippingLabels extends Component {
+	componentWillMount() {
+		this.setState( { expanded: this.isExpanded( this.props ) } );
+	}
+
+	componentWillReceiveProps( props ) {
+		this.setState( { expanded: this.isExpanded( props ) } );
+	}
+
+	isExpanded( { pristine, selectedPaymentMethod } ) {
+		return ! selectedPaymentMethod || ! pristine;
+	}
+
+	renderPlaceholder() {
+		return (
+			<div className="label-settings__placeholder">
+				<FormFieldSet>
+					<FormLabel className="label-settings__cards-label">
+						<span />
+					</FormLabel>
+					<FormSelect />
+				</FormFieldSet>
+				<FormFieldSet>
+					<FormLabel
+						className="label-settings__cards-label">
+						<span />
+					</FormLabel>
+					<p className="label-settings__credit-card-description">
+					</p>
+					<PaymentMethod selected={ false } isLoading={ true } />
+					<PaymentMethod selected={ false } isLoading={ true } />
+					<Button href="https://wordpress.com/me/billing" target="_blank" compact />
```

Comment:
Can we do a relative link here so that if you are testing on wpcalypso, dev, or another environment, you are not redirected to staging/production?